pandas/plotting/_matplotlib/converter.py

```diff
@@ -1097,6 +1097,8 @@ def __call__(self, x, pos=0):
             return ""
         else:
             fmt = self.formatdict.pop(x, "")
+            if isinstance(fmt, np.bytes_):
```

Comment:
Is this supposed to be in this PR?

Reply:
Yes, without it the call to strftime below fails bc strftime is now typed to want a str